root/edit/details/AddMedium.js

```diff
@@ -0,0 +1,191 @@
+/*
+ * @flow
+ * Copyright (C) 2020 MetaBrainz Foundation
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+import * as React from 'react';
+
+import MediumTracklist
+  from '../../medium/MediumTracklist';
+import {artistCreditsAreEqual}
+  from '../../static/scripts/common/immutable-entities';
+import DescriptiveLink
+  from '../../static/scripts/common/components/DescriptiveLink';
+import ExpandedArtistCredit
+  from '../../static/scripts/common/components/ExpandedArtistCredit';
+import loopParity from '../../utility/loopParity';
+
+type CondensedTrackACsRowProps = {
+  +artistCredit: ArtistCreditT,
+  +endNumber?: string,
+  +rowCounter: number,
+  +startNumber: string,
+};
+
+type CondensedTrackACsProps = {
+  +tracks?: $ReadOnlyArray<TrackT>,
+};
+
+type AddMediumEditT = {
+  ...EditT,
+  +display_data: {
+    +format: MediumFormatT | null,
+    +name?: string,
+    +position: number | string,
+    +release?: ReleaseT,
+    +tracks?: $ReadOnlyArray<TrackT>,
+  },
+};
+
+type Props = {
+  +allowNew?: boolean,
+  +edit: AddMediumEditT,
+};
+
+const CondensedTrackACsRow = ({
+  artistCredit,
+  endNumber,
+  rowCounter,
+  startNumber,
+}: CondensedTrackACsRowProps): React.Element<'tr'> => (
+  <tr className={loopParity(rowCounter)}>
+    <td className="pos t">
+      {endNumber && endNumber !== startNumber
+        ? startNumber + '-' + endNumber
+        : startNumber}
+    </td>
+    <td>
+      <ExpandedArtistCredit artistCredit={artistCredit} />
+    </td>
+  </tr>
+);
+
+const CondensedTrackACs = ({
+  tracks,
+}: CondensedTrackACsProps):
+  Array<React.Element<typeof CondensedTrackACsRow>> => {
+  if (!tracks) {
+    return [];
+  }
+
+  let thisCredit;
+  let thisPosition = 0;
+  let rowCounter = 1;
```

Comment:
Odd/even row classes have been shifted by one row, this fixes it:

```suggestion
  let rowCounter = 0;
```